libexec/cli/bootstrap.exec

```diff
@@ -47,7 +47,7 @@ while true; do
             SINGULARITY_NOTEST=1
             export SINGULARITY_NOTEST
         ;;
-        -f|--force)
+        -f|-F|--force)
             shift
```

Comment:
lol, so I might use CAPITAL F when I'm REALLY ANGRY AT BOOTSTRAP.

Reply:
lol.  It's just for consistency.  In other commands the short version of `--force` is `-F`, so I don't know why it was lowercase to begin with.